src/sentry/db/models/base.py

```diff
@@ -106,8 +106,25 @@ def get_relocation_scope(self) -> RelocationScope:
         Retrieves the `RelocationScope` for a `Model` subclass. It generally just forwards `__relocation_scope__`, but some models have instance-specific logic for deducing the scope.
         """
 
+        if isinstance(self.__relocation_scope__, set):
+            raise ValueError(
+                "Must define `get_relocation_scope` override if using multiple relocation scopes."
+            )
+
         return self.__relocation_scope__
 
+    @classmethod
+    def get_possible_relocation_scopes(cls) -> RelocationScope:
```

Comment:
ah, saw this function in the other PR, I feel like some context that I see in this PR is missing in the other one. Such as, `__relocation_scope__` can now be a set of `RelocationScope`

https://github.com/getsentry/sentry/pull/56268